apps/user-portal/src/api/multi-factor-fido.ts

```diff
@@ -94,8 +91,8 @@ export const startFidoFlow = (): Promise<any> => {
             "Access-Control-Allow-Origin": GlobalConfig.clientHost,
             "Content-Type": "application/x-www-form-urlencoded"
         },
-        method: HttpMethods.GET,
-        params: { appId: window.location.origin },
+        method: HttpMethods.POST,
+        data: {appId: window.location.origin},
```

Comment:
shall we add a space after the opening curly bracket and a space before the closing curly bracket?

Reply:
done